### PR TITLE
Add OpenInference instrumentation packages to the registry

### PR DESCRIPTION
**Pull request by mikeldking**
This PR adds registry entries for the JS and Python instrumentation packages created and maintained by Arize AI's OSS team.

### CODE REVIEW

File: data/registry/instrumentation-js-openai.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+language: javascript
+tags:
+  - javascript
+  - instrumentation
+  - openai
+urls:
+  repo: https://github.com/Arize-ai/openinference/tree/main/js/packages/openinference-instrumentation-openai
+  docs: https://github.com/Arize-ai/openinference/tree/main/js/packages/openinference-instrumentation-openai
+license: Apache 2.0 
+description: Instrumentation for tracing OpenAI NodeJS SDK
```
Comment (mikeldking) [inline data/registry/instrumentation-js-openai.yml:13]:
```suggestion
description: Instrumentation for tracing the OpenAI NodeJS SDK
```

File: data/registry/instrumentation-python-mistralai-openinference.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+language: python
+tags:
+  - python
+  - instrumentation
+  - dspy
+urls:
+  repo: https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-mistralai
+  docs: https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-mistralai
+license: Apache 2.0 
+description: Instrumentation for tracing Mistral AI framework
```
Comment (mikeldking) [inline data/registry/instrumentation-python-mistralai-openinference.yml:13]:
```suggestion
description: Instrumentation for tracing the Mistral AI python SDK
```

File: data/registry/instrumentation-python-openai-openinference.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+language: python
+tags:
+  - python
+  - instrumentation
+  - open ai
+urls:
+  repo: https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-openai
+  docs: https://github.com/Arize-ai/openinference/tree/main/python/instrumentation/openinference-instrumentation-openai
+license: Apache 2.0 
+description: Instrumentation for tracing Open AI
```
Comment (mikeldking) [inline data/registry/instrumentation-python-openai-openinference.yml:13]:
```suggestion
description: Instrumentation for tracing the OpenAI Python SDK
```

File: data/registry/instrumentation-python-openai-openinference.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore openai
+title: Open AI Python instrumentation
+registryType: instrumentation
+language: python
+tags:
+  - python
+  - instrumentation
+  - open ai
```
Comment (mikeldking) [inline data/registry/instrumentation-python-openai-openinference.yml:8]:
```suggestion
  - openai
```

File: data/registry/instrumentation-js-langchain.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# cSpell:ignore langchain
+title: LangChain Javascript instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-js-langchain.yml:2]:
```suggestion
title: LangChain Javascript Instrumentation
```

File: data/registry/instrumentation-js-openai.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# cSpell:ignore openai
+title: OpenAI Javascript instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-js-openai.yml:2]:
```suggestion
title: OpenAI Javascript Instrumentation
```

File: data/registry/instrumentation-python-dspy.yml [hunk 7]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore DSPy
+title: DSPy Python instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-python-dspy.yml:2]:
```suggestion
title: DSPy Python Instrumentation
```

File: data/registry/instrumentation-python-langchain-openinference.yml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore langchain
+title: LangChain Python instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-python-langchain-openinference.yml:2]:
```suggestion
title: LangChain Python Instrumentation
```

File: data/registry/instrumentation-python-llamaindex-openinference.yml [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore llamaindex
+title: LlamaIndex Python instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-python-llamaindex-openinference.yml:2]:
```suggestion
title: LlamaIndex Python Instrumentation
```

File: data/registry/instrumentation-python-mistralai-openinference.yml [hunk 10]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore Mistral AI
+title: Mistral AI Python instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-python-mistralai-openinference.yml:2]:
```suggestion
title: Mistral AI Python Instrumentation
```

File: data/registry/instrumentation-python-openai-openinference.yml [hunk 11]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore openai
+title: Open AI Python instrumentation
```
Comment (mikeldking) [inline data/registry/instrumentation-python-openai-openinference.yml:2]:
```suggestion
title: Open AI Python Instrumentation
```

**Comment by mikeldking** [comment]
@cartermp thanks so much for the review! cSpell is fixed.

File: data/registry/instrumentation-python-openai-openinference.yml [hunk 12]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore openai
```
Comment (cartermp) [inline data/registry/instrumentation-python-openai-openinference.yml:1]:
```suggestion
# cSpell:ignore openai Arize
```

File: data/registry/instrumentation-python-mistralai-openinference.yml [hunk 13]
```diff
@@ -0,0 +1,17 @@
+# cSpell:ignore Mistral mistralai openinference arize
```
Comment (cartermp) [inline data/registry/instrumentation-python-mistralai-openinference.yml:1]:
```suggestion
# cSpell:ignore Mistral mistralai openinference arize dspy
```

File: data/registry/instrumentation-js-openai.yml [hunk 14]
```diff
@@ -0,0 +1,20 @@
+# cSpell:ignore openai OpenAI Arize openinference
```
Comment (cartermp) [inline data/registry/instrumentation-js-openai.yml:1]:
```suggestion
# cSpell:ignore openai OpenAI Arize openinference arizeai
```

File: data/registry/instrumentation-js-langchain.yml [hunk 15]
```diff
@@ -0,0 +1,20 @@
+# cSpell:ignore langchain openinference arize Arize
```
Comment (cartermp) [inline data/registry/instrumentation-js-langchain.yml:1]:
```suggestion
# cSpell:ignore langchain openinference arize Arize arizeai
```

**Review by cartermp — COMMENTED** [review]
Looks like the checker had a few more things it was unhappy with, heh. This should cover it.

File: data/registry/instrumentation-python-mistralai-openinference.yml (hunk 13 above)
Comment (mikeldking) [reply data/registry/instrumentation-python-mistralai-openinference.yml:1]: oh this is wrong in the tag, thanks!

File: data/registry/instrumentation-python-mistralai-openinference.yml [hunk 16]
```diff
@@ -0,0 +1,17 @@
...
+title: Mistral AI Python Instrumentation
+registryType: instrumentation
+language: python
+tags:
+  - python
+  - instrumentation
+  - dspy
```
Comment (mikeldking) [inline data/registry/instrumentation-python-mistralai-openinference.yml:8]:
```suggestion
  - mistralai
```

**Comment by mikeldking** [comment]
@cartermp thanks again for the spelling review 🙏🏻

**Review by theletterf — APPROVED** [review]

**Pull request merged**